config/initializers/assets.rb

```diff
@@ -7,8 +7,3 @@
 # Rails.application.config.assets.paths << Emoji.images_path
 # Add Yarn node_modules folder to the asset load path.
 Rails.application.config.assets.paths << Rails.root.join('node_modules')
-
-# Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in the app/assets
-# folder are already added.
-Rails.application.config.assets.precompile += ['*.js', 'print-songs.css']
```

Comment:
Could you give some more context on this change? It actually seems sort of important

Reply:
we upgraded sprocket from 3 to 4, and finding asset files is the job of `manifest.js` which I added in this PR.
see https://github.com/rails/sprockets/blob/main/UPGRADING.md